test/unit/spec/OktaSignIn_spec.js

```diff
@@ -12,6 +12,7 @@ import Q from 'q';
 import $sandbox from 'sandbox';
 import Logger from 'util/Logger';
 import Widget from 'widget/OktaSignIn';
+import DeviceEnrollmentTerminalViewForm from '../helpers/dom/v2/DeviceEnrollmentTerminalViewForm';
```

Comment:
We don't need to test this in here.. Just add a test that can confirm that if proxyIdxResponse exists in the config as parameter then it bootstrap V2 router or not.. that's it.!